packages/types/src/redirects.ts

```diff
@@ -1,5 +1,23 @@
 import type { OAuthStrategy, SamlStrategy } from './strategies';
 
+export type AfterSignOutUrl = {
+  /**
+   * Full URL or path to navigate after successful sign out.
+   */
+  afterSignOutUrl?: string | null;
+};
+
+/**
+ * Redirect URLs for different actions.
+ * Mainly used to be used to type internal Clerk functions.
+ */
+export type RedirectOptions = {
+  signInForceRedirectUrl?: string | null;
+  signInFallbackRedirectUrl?: string | null;
+  signUpForceRedirectUrl?: string | null;
+  signUpFallbackRedirectUrl?: string | null;
+};
```

Comment:
Wouldn't it be safer to do 
```suggestion
export type RedirectOptions = SignUpForceRedirectUrl & SignUpFallbackRedirectUrl & SignInFallbackRedirectUrl & SignInForceRedirectUrl
```

Reply:
Nice one, thanks!